comparator/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'org.mobilitydata'
+version "${System.getenv("versionTag")}"
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.google.api-client:google-api-client:1.31.2'
```

Comment:
I believe the `compile` and `testCompile` here should be `implementation` and `testImplementation`?

Reply:
Yes!